tools/topology/topology1/sof-tgl-max98373-rt5682.m4

```diff
@@ -85,7 +85,7 @@ define(DMIC_PIPELINE_16k_ID, `9')
 define(DMIC_PIPELINE_KWD_ID, `10')
 define(DMIC_DAI_LINK_16k_ID, `2')
 # Offload DMIC_PIPELINE_48K to secondary core of TGL.
-define(DMIC_PIPELINE_48k_CORE_ID, 1)
+ifdef(`IGO', `define(`DMIC_PIPELINE_48k_CORE_ID', 1)')
```

Comment:
Comment should be updated too, since the condition is now based on whether IGO is enabled.

Reply:
Ack, cc @sathya-nujella @RDharageswari

Reply:
would there be any drawback if we assigned the DMIC_48 to core1 always?

That would make sure the Intel CI/daily tests could catch multi-core issues that otherwise cannot be reproduced.

Reply:
from CI perspective to catch MC issues seems ok, but from product perspectives appears risky IMHO.

There are many products using this topology, if they assume no issues in moving to Core-1 it causes problems.
@yongzhi1 mentioned, moving DMIC48K moving to core-1 caused this issue: https://github.com/thesofproject/sof/issues/4382

IMHO,
https://github.com/thesofproject/sof/issues/4382 needs to be solved, but this PR helps to isolate the risk for products in use already.

Reply:
@sathya-nujella I would imagine that 'products in use already' use their own branch? Is it really necessary to protect this?

Reply:
Hi @plbossart , All the products are using the google_private branch which is based on our public tgl-<product branches> (e.g. sof/tree/tgl-012-drop-stable).

In that case, as you are thinking: good to take this change to public sof tgl-product branch (which can then be picked to private product branch).

@yongzhi1 
Requesting you to post this change on to tgl-013-drop-stable as per above inputs from Pierre.